packages/zoe/test/unitTests/contractSupport/test-ratio.js

```diff
@@ -11,7 +11,9 @@ import {
   multiplyBy,
   divideBy,
   invertRatio,
+  oneMinus,
 } from '../../../src/contractSupport';
+import { multiplyRatios, addRatios } from '../../../src/contractSupport/ratio';
```

Comment:
small thing, but can we import these from contractSupport as well?

Reply:
done